stwfsapy/frequency_features.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 Leibniz Information Centre for Economics
```

Comment:
```suggestion
# Copyright 2020 Leibniz Information Centre for Economics
```
```suggestion
# Copyright 2021 Leibniz Information Centre for Economics
```
Or 2020-2021. Also in the other files.

Reply:
I have changed it for the new files in this PR. Will do the remaining ones after.